schema-registry/rest-service/src/test/java/com/hortonworks/registries/schemaregistry/avro/util/AvroSchemaRegistryClientUtil.java

```diff
@@ -13,55 +13,27 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  **/
-package com.hortonworks.registries.schemaregistry.avro;
+package com.hortonworks.registries.schemaregistry.avro.util;
 
+import com.hortonworks.registries.schemaregistry.avro.AvroSchemaRegistryClientTest;
 import com.hortonworks.registries.serdes.Device;
-import com.hortonworks.registries.schemaregistry.client.SchemaRegistryClient;
-import com.hortonworks.registries.webservice.RegistryApplication;
-import com.hortonworks.registries.webservice.RegistryConfiguration;
-import io.dropwizard.testing.ResourceHelpers;
-import io.dropwizard.testing.junit.DropwizardAppRule;
 import org.apache.avro.Schema;
 import org.apache.avro.generic.GenericData;
 import org.apache.avro.generic.GenericRecord;
 import org.apache.avro.specific.SpecificData;
 import org.apache.avro.specific.SpecificRecord;
 import org.junit.Assert;
-import org.junit.Before;
-import org.junit.ClassRule;
-import org.junit.Rule;
-import org.junit.rules.TestName;
-
 import java.io.IOException;
 import java.io.InputStream;
-import java.util.Collections;
 import java.util.Date;
-import java.util.Map;
 import java.util.Random;
 
 /**
  *
  */
-public abstract class AbstractAvroSchemaRegistryCientTest {
-
-    @ClassRule
-    public static final DropwizardAppRule<RegistryConfiguration> DROPWIZARD_APP_RULE
-            = new DropwizardAppRule<>(RegistryApplication.class, ResourceHelpers.resourceFilePath("schema-registry-test.yaml"));
-
-    protected final String rootUrl = String.format("http://localhost:%d/api/v1", DROPWIZARD_APP_RULE.getLocalPort());
-    protected final Map<String, String> SCHEMA_REGISTRY_CLIENT_CONF = Collections.singletonMap(SchemaRegistryClient.Configuration.SCHEMA_REGISTRY_URL.name(), rootUrl);
-
-    @Rule
-    public TestName TEST_NAME_RULE = new TestName();
-
-    protected SchemaRegistryClient schemaRegistryClient;
-
-    @Before
-    public void setup() throws IOException {
-        schemaRegistryClient = new SchemaRegistryClient(SCHEMA_REGISTRY_CLIENT_CONF);
-    }
+public class AvroSchemaRegistryClientUtil {
```

Comment:
nit: it can be made final and have private constructor so that this can never be instantiated or extended.